src/Hooks/addUser.jsx

```diff
@@ -0,0 +1,15 @@
+import userService from '../Services/user';
```

Comment:
Since you used absolute paths, why do you still use relative paths?
```suggestion
import userService from 'Services/user';
```